base_tier_validation_forward/models/tier_validation.py

```diff
@@ -18,6 +19,16 @@ def _compute_can_forward(self):
             definitions = reviews.mapped("definition_id")
             rec.can_forward = True in definitions.mapped("has_forward")
 
+    def _compute_can_backward(self):
+        for rec in self:
+            if not rec.can_review:
+                rec.can_backward = False
+                continue
+            sequences = self._get_sequences_to_approve(self.env.user)
+            reviews = rec.review_ids.filtered(lambda l: l.sequence in sequences)
+            definitions = reviews.mapped("definition_id")
+            rec.can_backward = True in definitions.mapped("backward")
```

Comment:
\





```suggestion
    def _compute_can_backward(self):
        for rec in self:
            if not rec.can_review:
                rec.can_backward = False
                continue
            sequences = self._get_sequences_to_approve(self.env.user)
            reviews = rec.review_ids.filtered(lambda l: l.sequence in sequences)
            definitions = reviews.mapped("definition_id")
            rec.can_backward = True in definitions.mapped("backward")
```

Should we merge this into `_compute_can_forward`? They are almost identical. May be we can change the function name to `_compute_forward_backward`?

Reply:
Yes, I agree and have updated the code